<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Win Mail Tester</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../data/icons.qrc">
    <normaloff>:/icons/64x64/internet-mail.png</normaloff>:/icons/64x64/internet-mail.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="5" column="0">
        <widget class="QLabel" name="label_attachment">
         <property name="text">
          <string>Attachment</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QWidget" name="widget_attachment" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="attachment"/>
          </item>
          <item>
           <widget class="QPushButton" name="button_select_attachment">
            <property name="text">
             <string>Select</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_subject">
         <property name="text">
          <string>Subject</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_to_addr">
         <property name="text">
          <string>To address</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="subject"/>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="to_addr"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_to_name">
         <property name="text">
          <string>To name</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="to_name"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="layout_select_file"/>
    </item>
    <item>
     <widget class="QTextBrowser" name="text_body">
      <property name="textInteractionFlags">
       <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextEditable|Qt::TextSelectableByMouse</set>
      </property>
      <property name="openExternalLinks">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Preferred</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="layout_buttons">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="button_test_mapi">
        <property name="text">
         <string>Test MAPI</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_test_outlook">
        <property name="text">
         <string>Test Outlook</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_exit">
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>437</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="action_about"/>
    <addaction name="action_about_qt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="action_settings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="action_about_qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>to_name</tabstop>
  <tabstop>to_addr</tabstop>
  <tabstop>subject</tabstop>
  <tabstop>attachment</tabstop>
  <tabstop>button_select_attachment</tabstop>
  <tabstop>text_body</tabstop>
  <tabstop>button_test_mapi</tabstop>
  <tabstop>button_test_outlook</tabstop>
  <tabstop>button_exit</tabstop>
 </tabstops>
 <resources>
  <include location="../data/icons.qrc"/>
 </resources>
 <connections/>
</ui>
